### PR TITLE
JS_Week1_Homework_Enis

### PR DESCRIPTION
**Pull request by okayns**

### CODE REVIEW

File: Week1/homework/10_compare_Arrays.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+let cars = ['tesla', 'ferrari', 'audi', 'opel', 'mercedes', 'bmw', 'murat131'];
+
```
Comment (unmeshvrije) [inline Week1/homework/10_compare_Arrays.js]: stray new line.

File: Week1/homework/1_helloWorld.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+let helloArray = ['Merhaba Dünya // Turkish',
+    'Hallo Wereld // Dutch', 'Hai Dunia // Malay', 'Labas Pasauli // Lithuanian', 'Hei Maailma //Finnish', 'Γειά σου Κόσμε // Greek', 'Pozdravljen, svet // Slovenian', 'Ndewo Ụwa // Igbo', 'Halo Dunia // Indonesian', 'Здраво Свете // Serbian'
+];
+
+console.log(helloArray[0]);
```
Comment (unmeshvrije) [inline Week1/homework/1_helloWorld.js]: you could have simply given string arguments to the log() function.
Having an array is not necessary for this assignment. It is good for your practice though.

File: Week1/homework/2_error_Debug.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+/* console.log('I'm awesome'); this code gives SyntaxError: missing ) after argument listwill because we need to escape the single quotation
```
Comment (unmeshvrije) [inline Week1/homework/2_error_Debug.js]: Nice

File: Week1/homework/3_log_Number.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+let numberX;
+
+console.log('numberX is an undefined number for now, just wait for a sec');
```
Comment (unmeshvrije) [inline Week1/homework/3_log_Number.js]: dramatic console log! I like it. Keep it up until JavaScript 3 module :)

File: Week1/homework/3_log_Number.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+let numberX;
+
+console.log('numberX is an undefined number for now, just wait for a sec');
+
+console.log(numberX);
+
```
Comment (unmeshvrije) [inline Week1/homework/3_log_Number.js]: Too many newlines. when you start writing bigger code, you will realize that it is annoying to see a new line between every line.

File: Week1/homework/4_log_String.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+let myString = 'Enis OKAY';
+
+console.log('This is my Full Name');
+
+console.log(myString);
+
+let = 'Deren Busra';
+
+console.log('This is my daughter`s name');
```
Comment (unmeshvrije) [inline Week1/homework/4_log_String.js]: Backtick (`) is not an apostrophe ! that is a function of a single quote. You know how to escape it

File: Week1/homework/8_type_Checker.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    eyeColor: 'black',
+};
+
+let favAnimals = ['Cat', 'Dog'];
+
+console.log(typeof a);
+
+console.log(typeof contactInfo);
+
+// I used TERNARY OPERATOR instead of if statement. Because this way is more efficient in this case.
```
Comment (unmeshvrije) [inline Week1/homework/8_type_Checker.js]: well done!

File: Week1/homework/9_log_remainder.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+let x = 7;
+x = x % 3;
+
+let y = 21;
+y = y % 4;
+
+let z = 13;
+z = z % 2;
+
```
Comment (unmeshvrije) [inline Week1/homework/9_log_remainder.js]: you are supposed to write the values in the comments with explanation as to how you reached that conclusion.

File: Week1/homework/4_log_String.js (hunk 6 above)
Comment (okayns) [reply Week1/homework/4_log_String.js]: I didn't realize that. (`) just an old habit for me.
Correct way - - - >>> console.log('This is my daughter\'s name');

File: Week1/homework/3_log_Number.js (hunk 5 above)
Comment (okayns) [reply Week1/homework/3_log_Number.js]: Yes absolutely you are right. I just think that can be more readable for teacher. After this week i will clear empty lines.

File: Week1/homework/1_helloWorld.js (hunk 2 above)
Comment (okayns) [reply Week1/homework/1_helloWorld.js]: I wanted to practice a little bit, as you said. I thought a little bit difference can be good 😃

**Comment by okayns** [comment]
@unmeshvrije  Thank you for feedbacks. I will pay attention to those advices for next weeks

**Pull request closed without merging**